G/gdk_pixbuf/build_tarballs.jl

```diff
@@ -78,7 +78,7 @@ dependencies = [
     Dependency("Glib_jll"; compat="2.68.3"),
```

Comment:
Setting Glib_jll compat to 2.76.5 gets x86_64-linux-gnu compiling for me locally. I think it's related to GObject introspection, which is enabled just for that platform.